src/index.html

```diff
@@ -2,17 +2,61 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0" />
     <title>Stars</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="./styles/stars.css"
+    />
```

Comment:
Keep your attributes correctly formatted. Since the `<link>` element has more than 2 attributes, each attribute should start on a new line with 2-space indentation relative to the tag. The tag's closing bracket should be on the same level as the opening one.

Good example:
<link
  rel="stylesheet"
  href="./styles/stars.css"
>